docs/user/dql/functions.rst

```diff
@@ -341,13 +341,13 @@ Return type: DOUBLE
 
 Example::
 
-    os> SELECT DEGREES(1.57)
+    os> SELECT DEGREES(0)
     fetched rows / total rows  = 1/1
-    +-------------------+
-    | DEGREES(1.57)     |
-    |-------------------|
-    | 89.95437383553924 |
-    +-------------------+
+    +--------------+
+    | DEGREES(0)   |
+    |--------------|
+    | 0.0          |
+    +--------------+
```

Comment:
could you describe a bit what the issue is in PR description?

Reply:
Done